admin/class-yoast-notification.php

```diff
@@ -38,8 +44,10 @@ class Yoast_Notification {
 	 * @param array  $options Set of options.
 	 */
 	public function __construct( $message, $options = array() ) {
-		$this->options         = wp_parse_args( $options, $this->defaults );
-		$this->message         = $message;
+		$this->message = $message;
+		$this->options = wp_parse_args( $options, $this->defaults );
+
+		$this->verify_options();
```

Comment:
I think it's better to consolidate this line and the previous into one line stating:
```
$this->options = $this->normalize_options( $options );
```
And then `normalize_options` method then uses the defaults. Then we have only 1 line in the constructor dealing with the `$options`.